src/networks.json

```diff
@@ -914,6 +914,62 @@
     "start": 536483,
     "imageIPFS": "QmeS75uS7XLR8o8uUzhLRVYPX9vMFf4DXgKxQeCzyy7vM2"
   },
+  "60001": {
+    "key": "60001",
+    "name": "Thinkium Testnet Chain 1",
+    "shortName": "ThinkiumTest1",
+    "chainId": 60001,
+    "network": "thinkiumtest1",
+    "multicall": "0xc49bc485d4f943b287edadbce45eb1a1220ffdfe",
```

Comment:
How to check the source code of this address in this network? 🤔 

I go here https://test1.thinkiumscan.net/address/0xc49bc485d4f943b287edadbce45eb1a1220ffdfe and see something else but not a multicall, this chain work differently?

Reply:
wait 5 seconds , you will see the link is available. This address is the multicall contract I'v deployed. I'm very sure our chain works well. Pls check again.